tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -29,6 +29,27 @@ Feature: full text search
       | spaces           |
 
 
+  Scenario Outline: search folder using a tag
+    Given using <dav-path-version> DAV path
+    And user "Alice" has created folder "uploadFolder1"
+    And user "Alice" has created folder "uploadFolder2"
+    And user "Alice" has created folder "uploadFolder3"
+    And user "Alice" has created the following tags for folder "uploadFolder1" of the space "Personal":
+      | tag1 |
+    And user "Alice" has created the following tags for folder "uploadFolder2" of the space "Personal":
+      | tag1 |
+    When user "Alice" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
```

Comment:
```suggestion
    And the search result of user "Alice" should contain only these folders:
```
OR
   ` And the search result of user "Alice" should contain only these resources:`

Reply:
okay